### DISCUSSION TITLE
Welcome to VirtualGimbal Discussions!

### DISCUSSION THREAD

**Discussion by yossato** [yossato/virtualGimbal#27] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mcy260** [comment]
where do i buy it? I mean the device, it seems the prototype is from 2016, but now in 2023 where can I buy it?
